docs/_posts/2019-12-31-ace3-version3130.md

```diff
@@ -0,0 +1,117 @@
+---
+title: ACEREP  &#35;00010
+description: Status report on ACE3 version 3.13.0
+parent: posts
+image: /img/news/191231_medical1.jpg
+author: alganthe, mharis001
+layout: post
+---
+
+This post marks the release of ACE3 v3.13.0 and, as a result, the public release of the medical rewrite. The [previous news post]({{site.baseUrl}}/2019/12/18/medical-rewrite.html) provided perspective on the guiding principles and goals of the medical rewrite. In this post, we detail the key changes of the rewrite in order to give users a better starting point as they explore ACE3's new medical system.
+
+<!--more-->
+
+As the name suggests, the medical rewrite was a complete overhaul of the medical system. The majority of the changes and improvements were made to the code base and as a result will not be noticeable to end users. However, a better code base sets the stage for easier and faster development of the system in the future, opening possibilities for more features.
```

Comment:
```suggestion
As the name suggests, the medical rewrite was a complete overhaul of the medical system. The majority of the changes and improvements were made to the code base and as a result will not be always noticeable to end users. However, a better code base sets the stage for easier and faster development of the system in the future, opening possibilities for more features.
```

Reply:
Moved the "always" before the "be". Sounds a bit better I think.